pkg/scheduler/actions/preempt/preempt.go

```diff
@@ -240,7 +244,8 @@ func preempt(
 
 		for !victimsQueue.Empty() {
 			// If reclaimed enough resources, break loop to avoid Sub panic.
-			if preemptor.InitResreq.LessEqual(node.FutureIdle(), api.Zero) {
+			// If preemptor's queue is overused, it means preemptor can not be allcated. So no ned care about the node idle resourace
```

Comment:
Tips: no **need** care about the node idle resource. But it doesn't matter.

Reply:
`preemptor.InitResreq.LessEqual(node.FutureIdle(), api.Zero)` is old logic, I think it want to handle some pod are in `terminating` case. This node is future idle for preemptor then break preempt logic.

Reply:
> `preemptor.InitResreq.LessEqual(node.FutureIdle(), api.Zero)` is old logic, I think it want to handle some pod are in `terminating` case. This node is future idle for preemptor then break preempt logic.

I mean the wrong spell in the comments.

Reply:
：） fixed typos